src/ui.form-checkbox-multi-state/useAttrs.ts

```diff
@@ -5,26 +5,29 @@ import defaultConfig from "./config.ts";
 import type { Ref } from "vue";
 import type { UseAttrs } from "../types.ts";
 import type { UCheckboxMultiStateProps, Config } from "./types.ts";
+import type { UCheckboxProps } from "../ui.form-checkbox/types.ts";
 
 type ComponentState = {
   selected: Ref<{ icon?: string }>;
 };
 
+type MultiStateCheckboxAttrs = {
+  value?: Partial<UCheckboxProps["value"]>;
+  config?: {
+    defaults?: {
+      checkedIcon?: string;
```

Comment:
this also contains in UCheckboxProps. No need to define this type in already included indisde the `UCheckboxProps`